components/Dashboard/PanelGroup.vue

```diff
@@ -1,130 +1,108 @@
-      <div class="card-panel" @click="statusI = !statusI,status=false,statusF=false,statusD=false " >
-        <div class="card-panel-icon-wrapper icon-message" >
-            <img src="@/assets/img/icons8-workspace-64.png" class="card-panel-icon" />
+      </el-col>
+
+      <el-col :xs="12" :sm="12" :lg="6" class="card-panel-col">
+        <div
+          class="card-panel"
+          @click="statusD = !statusD,statusI=false,statusF=false,status=false"
+        >
+          <div class="card-panel-icon-wrapper icon-shopping">
+            <img src="@/assets/img/icons8-password-64.png" alt>
+          </div>
+          <div class="card-panel-description">
+            <div class="card-panel-text">Nondelivery</div>
+            <count-to :start-val="0" :end-val="500" :duration="3200" class="card-panel-num"/>
+          </div>
         </div>
-
-
-        <div class="card-panel-description"  >
-          <div class="card-panel-text">Workpiece</div>
-          <count-to :start-val="0" :end-val="500" :duration="3000" class="card-panel-num"/>
+      </el-col>
+
+      <el-col :xs="12" :sm="12" :lg="6" class="card-panel-col">
+        <div
+          class="card-panel"
+          @click="statusF = !statusF,statusI=false,status=false,statusD=false"
+        >
+          <div class="card-panel-icon-wrapper icon-shopping">
+            <img src="@/assets/img/icons8-job-seeker-64.png" class="card-panel-icon">
+          </div>
+          <div class="card-panel-description">
+            <div class="card-panel-text">Delivered</div>
+            <count-to :start-val="0" :end-val="500" :duration="3600" class="card-panel-num"/>
+          </div>
         </div>
-      </div>
-    </el-col>
-
-
-
-    <el-col :xs="12" :sm="12" :lg="6" class="card-panel-col">
-      <div class="card-panel" @click="statusD = !statusD,statusI=false,statusF=false,status=false">
-        <div class="card-panel-icon-wrapper icon-shopping" >
-          <img src="@/assets/img/icons8-password-64.png" alt=""/>
-        </div>
-        <div class="card-panel-description"  >
-          <div class="card-panel-text">Nondelivery</div>
-          <count-to :start-val="0" :end-val="500" :duration="3200" class="card-panel-num"/>
-        </div>
-      </div>
-    </el-col>
-    
-    <el-col :xs="12" :sm="12" :lg="6" class="card-panel-col">
-      <div class="card-panel" @click="statusF = !statusF,statusI=false,status=false,statusD=false">
-        <div class="card-panel-icon-wrapper icon-shopping">
-         
-          <img src="@/assets/img/icons8-job-seeker-64.png" class="card-panel-icon"/>
-         
-        </div>
-        <div class="card-panel-description">
-          <div class="card-panel-text">Delivered</div>
-          <count-to :start-val="0" :end-val="500" :duration="3600" class="card-panel-num"/>
-        </div>
-      </div>
-    </el-col>
-  </el-row>
-  
-
-<TransactionTable :status='status'/>
-
-<TransactionTableD :status='statusD' />
```

Comment:
The names `TransactionTableD`, `TransactionTableF` and `TransactionTableI` are not meaningful, you should spend more name giving names to your functions, components and variables.

Reply:
These components are deleted. Because i do not need these.